### ISSUE TITLE
always Restart at the boot animation 

### ISSUE THREAD

**Issue by lihonglinglory** [DEV-MSM8909/android_device_huawei_yax_lte#1]
..I compied cm12.1 for honor4a(huawei Y6)...i flash it and always Restart at the boot animation

**Comment by jarpii** [comment]
i am compile cm12.1 i have error restar al the bootanimation...

**Comment by lihonglinglory** [comment]
You can refer to the device for Kiwi,thanks

**Comment by Runner85sx** [comment]
Bootloop after flashing the rom. recovery works fine.
I could revive it. put dload to sdcard. adb reboot-bootloader hold down vol+&vol- and fastboot boot recovery-stock.img 

Now emui recovery will flash the stockrom.

**Comment by jarpii** [comment]
I recompiled cm12.1, but still I try not hope to know the error bootanimation

**Comment by Runner85sx** [comment]
https://github.com/DestructoSphere got it working.
I forked his repos and modified "android/.repo/local_manifests/local_manifest.xml" to fetch all needed sources.
https://github.com/Runner85sx/local-manifest

**Comment by jarpii** [comment]
Your Vendor properaty is correct?

**Comment by Runner85sx** [comment]
Yes. It boots fine. I only needed to replace updater_binary in zip.

**Comment by lihonglinglory** [comment]
well,i compiled  and flash it~~that camera not work

**Comment by Runner85sx** [comment]
Please flash https://mega.nz/#!H40XxQiT!CbBxbPVq7X5zuGNi3tRp_RE6KMyqex75J5A1ikGET7o
and gapps from: http://opengapps.org  I used arm, 5.1, nano

**Comment by lihonglinglory** [comment]
OK，thanks
